doc/whats_new/v0.22.rst

```diff
@@ -45,6 +45,15 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+:mod:`sklearn.inspection`
```

Comment:
```suggestion
:mod:`sklearn.feature_extraction`
```

please fix the headline length and put sections in alphabetical order

Reply:
Done